ui/components/AccountsNotificationPanel/AccountsNotificationPanelAccounts.tsx

```diff
@@ -67,44 +77,97 @@ function WalletTypeHeader({
   accountType,
   onClickAddAddress,
   walletNumber,
+  keyringId,
 }: {
   accountType: AccountType
   onClickAddAddress?: () => void
   walletNumber?: number
+  keyringId: string | null
 }) {
   const { t } = useTranslation()
   const { title, icon } = walletTypeDetails[accountType]
+  const dispatch = useBackgroundDispatch()
+  const keyringMetadata = useBackgroundSelector(
+    (state) => state.keyrings.keyringMetadata
+  )
+
+  const sectionCustomName = keyringId
+    ? keyringMetadata[keyringId].customName
```

Comment:
Let's make sure it won't throw when keyringMetadata will be undefined for some reason
```suggestion
    ? keyringMetadata[keyringId]?.customName
```